.github/workflows/build.yml

```
@@ -18,8 +19,26 @@ on:
       - '**.md'
 
 jobs:
+  check_build_safety:
+    name: Check if Safe to Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Emit success code
+        if: ${{ github.repository == 'liquibase/liquibase' || github.event.label.name == 'SafeToBuild') }}
+        run: |
+          echo "PR is safe to build"
+          exit 0 // add success code
+
+      - name: Emit failure code
+        if: ${{ github.repository != 'liquibase/liquibase' && github.event.label.name == 'SafeToBuild') }}
```

This statement looks incorrect to me. I think it should be something like:

```suggestion
        if: ${{ github.repository != 'liquibase/liquibase' || github.event.label.name != 'SafeToBuild') }}
```

Otherwise, no build will ever succeed, since adding the label to a non-liquibase PR will not get past this check.